src/rtcModule/webrtcPrivate.h

```diff
@@ -486,6 +494,39 @@ class RtMessageComposer: public chatd::Command
         updateLenField();
     }
 };
+
+class DnsRequest : public WebsocketsClient
+{
+private:
+    DnsRequest() {}
+    static DnsRequest* instance;
+
+public:
+    static DnsRequest* getInstance()
+    {
+        if (!instance)
+        {
+            instance = new DnsRequest();
+        }
+
+        return instance;
+    }
+
+    virtual ~DnsRequest() {}
```

Comment:
Shouldn't `instance` become `null`?? (the global variable).

Reply:
It's done at file webrtc.cpp line  50

Reply:
That's the initialization of the value. I mean at dtor. Anyway, there's no need of a Singleton here, simply add the `DnsRequest` as a member of `RtcModule`.